intl/en/messages.json

```diff
@@ -1,4 +1,6 @@
 {
-  "f93ce20d178e2d16f45ed0a1a117b990": "example complete",
-  "5bdcd0e36db5b1100819f9aa25e9bdf0": "Cloudant {{regex}} syntax does not support global"
+  "3cde8cc9bca22c67278b202ab0720106": "No instance with id {0} found for {1}",
+  "5bdcd0e36db5b1100819f9aa25e9bdf0": "Cloudant {{regex}} syntax does not support global",
+  "934c33143a59fd286ff6f7d0b8fb6875": "Invalid settings: \"url\" OR \"username\" AND \"password\" required",
+  "f93ce20d178e2d16f45ed0a1a117b990": "example complete"
```

Comment:
@b-admike This is where `example complete` is used. Does not necessarily need to be an error. https://github.com/strongloop/loopback-connector-cloudant/blob/2ba7d7f77a96203c617699c4a5c677f3880689ec/example/example.js#L39